addons/survey/static/src/js/survey_form.js

```diff
@@ -26,7 +26,12 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
         this.options = this.$target.find('form').data()
         var $timer = $('.o_survey_timer');
         if ($timer.length) {
-            this.surveyTimerWidget = new publicWidget.registry.SurveyTimerWidget(this);
+            var timeLimitMinutes = this.options['timeLimitMinutes'];
+            var timer = this.options['timer'];
+            this.surveyTimerWidget = new publicWidget.registry.SurveyTimerWidget(this, {
+                'timer': timer,
+                'timeLimitMinutes': timeLimitMinutes
+            });
             this.surveyTimerWidget.attachTo($timer);
```

Comment:
Not sure about that one but maybe do it in a promise as attachedTo is asynchronous ? Something like this:

return this._super.apply(this,argument).then(function () {
  return Promise.resolve(self.surveyTimerWidget.attachTo($timer));
});